snapshot/utils.go

```diff
@@ -19,3 +19,7 @@ func isNydusMetaLayer(labels map[string]string) bool {
 	_, ok := labels[label.NydusMetaLayer]
 	return ok
 }
+
+func isNydusLayer(labels map[string]string) bool {
```

Comment:
Should be removed?

Reply:
Let's close this PR. #387  is upgrading the deps in a elegant way